frontend/src/components/Dialogs/EnableTelegram.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useTheme } from '@mui/material/styles';
+import QRCode from 'react-qr-code';
+import {
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Button,
+} from '@mui/material';
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  tgBotName: string;
+  tgToken: string;
+  onClickEnable: () => void;
+}
+
+const EnableTelegramDialog = ({
+  open,
+  onClose,
+  tgBotName,
+  tgToken,
+  onClickEnable,
+}: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+
+  return (
+    <Dialog
+      open={open}
+      onClose={onClose}
+      aria-labelledby='enable-telegram-dialog-title'
+      aria-describedby='enable-telegram-dialog-description'
+    >
+      <DialogTitle id='open-dispute-dialog-title'>{t('Enable TG Notifications')}</DialogTitle>
+      <DialogContent>
+        <div style={{ textAlign: 'center' }}>
+          <QRCode
```

Comment:
Is it not possible to also include on click a redirection to bot's web page same as below?

Reply:
Do you mean open the `t.me` website also on click on the <QRcode />? Sounds good!